docs/start.rst

```diff
@@ -4,15 +4,15 @@
 Getting started
 ===============
 
-This tutorial is for bioinformaticians. It will help you install the ReSDK
-and explain some basic commands. We will connect to a Resolwe server,
-do some basic queries, and align raw reads to a genome.
+This tutorial is for bioinformaticians. It will help you install the ReSDK and explain some basic
```

Comment:
Please wrap lines at 80 characters. I know that some lines are longer, but they should be fixed when they are changed.